staging/src/k8s.io/client-go/examples/out-of-cluster-client-configuration/README.md

```diff
@@ -16,6 +16,7 @@ Make sure your `kubectl` is configured and pointed to a cluster. Run
 Run this application with:
 
     cd out-of-cluster-client-configuration
+    go get -v -u ./...
```

Comment:
this is not really true if you are on a release branch. Please add a comment that you have to use a vendoring tool to get the correct dependencies.